themes/default/content/blog/typescript-versions/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Typescript Versions"
```

Comment:
Consider a more descriptive and engaging title, something like Run Pulumi with Any Typescript Version